upi/vsphere/lb/haproxy.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=haproxy
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+TimeoutStartSec=0
+ExecStartPre=-/bin/podman kill haproxy
+ExecStartPre=-/bin/podman rm haproxy
+ExecStartPre=/bin/podman pull quay.io/openshift/origin-haproxy-router
+ExecStart=/bin/podman run --name haproxy \
+  --net=host \
+  --privileged \
+  --entrypoint=/usr/sbin/haproxy \
+  -v /etc/haproxy/haproxy.conf:/var/lib/haproxy/conf/haproxy.conf:Z \
+  quay.io/openshift/origin-haproxy-router -f /var/lib/haproxy/conf/haproxy.conf
```

Comment:
why is this file `-f` different from ??https://github.com/openshift/installer/pull/3429/commits/254c99c9e828706ececb9025d0b63347857d7984#diff-0a3feea2452df114fdd7fffd188e6f8eR8

Reply:
It hasn't been changed.  That is the path to the config in the container.